packaging/hudi-spark-bundle/pom.xml

```diff
@@ -127,6 +127,8 @@
                   <pattern>parquet.schema</pattern>
                   <shadedPattern>org.apache.hudi.parquet.schema</shadedPattern>
                 </relocation>
+                <!-- TODO: Revisit GH ISSUE #533 & PR#633-->
```

Comment:
@bvaradar this needs your special attention..  Had to do this to get hive sync as a part of spark datasource working..  some background in those issues and prs

Reply:
Ack. We would have to check with other certified combinations of Hive and Spark